datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxService.java

```diff
@@ -217,11 +218,15 @@ public Long getResourceIdByResourceName(String resourceName) {
 
     @VisibleForTesting
```

Comment:
It is visible for testing, then can we have a proper test for it, please?

Reply:
Yeah sure.

Reply:
Refactored the ExternalDatabase configuration logic to another class and wrote some unit tests for it.